src/test/integ/buffer-write-cmd.test.ts

```diff
@@ -84,4 +127,18 @@ describe("BufWriteCmd integration", () => {
         assert.equal(doc.getText(), "aaa");
         assert.equal(await readFile(doc.uri), "hello world");
     });
+
+    it("Writing to a file in a symbolically linked workspace should succeed", async () => {
```

Comment:
nit: tests don't need to mention "should succeed" or other redundant phrases. it's assumed that tests are testing correct behavior, except for rare cases where it's not obvious.
```suggestion
    it("Writing to a file in a symlinked workspace", async () => {
```

Reply:
Removed the redundant wording in the latest change.